crypto/pem/pem_lib.c

```diff
@@ -344,7 +345,13 @@ int PEM_ASN1_write_bio(i2d_of_void *i2d, const char *name, BIO *bp,
         }
     }
 
-    if ((dsize = i2d(x, NULL)) <= 0) {
+    dsize = 0;
```

Comment:
you init this above.. (minor nit)

Reply:
Sure. Moved the earlier init inside the if.